sda/internal/database/db_functions.go

```diff
@@ -635,14 +635,9 @@ func (dbs *SDAdb) getUserFiles(userID string) ([]*SubmissionFileInfo, error) {
 	db := dbs.DB
 
 	// select all files of the user, each one annotated with its latest event
-	const query = "SELECT f.submission_file_path, e.event, f.created_at " +
-		"FROM sda.files f " +
-		"LEFT JOIN ( " +
-		"SELECT DISTINCT ON (file_id) file_id, started_at, event " +
-		"FROM sda.file_event_log " +
-		"ORDER BY file_id, started_at DESC" +
-		") e ON f.id = e.file_id " +
-		"WHERE f.submission_user = $1; "
+	const query = "SELECT f.submission_file_path, e.event, f.created_at FROM sda.files f " +
```

Comment:
This function (`getUserFiles`) is also used in the `/files` enpoint, right? Do we really want these changes to that request too?

Reply:
Yes, because files that are already part of a dataset should be of no interest.